src/mailing/templates/orderCompleted/html.ejs

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Email Template</title>
+    <link rel='stylesheet' href='./style.css' data-inline>
```

Comment:
In `sender.js` when we set up the `baseEmail` configuration options, we tell `juice` (the library that handles parsing the CSS for these emails) to use `'src/mailing/templates'` as the base path when looking for CSS files (or other web resources) that are asked for by the HTML. So we need to link this stylesheet, relative to that path.

```suggestion
    <link rel='stylesheet' href='./orderCompleted/style.css' data-inline>
```

Reply:
@akosasante thank you so much!!!!!